astropy/io/fits/diff.py

```diff
@@ -29,7 +29,7 @@
 from .card import Card, BLANK_CARD
 from .header import Header
 # HDUList is used in one of the doctests
-from .hdu.hdulist import fitsopen, HDUList  # pylint: disable=W0611
```

Comment:
Oh, that explains why I had never seen this before: 125ad384d04b04d612eb11ffb149ef682cb1726c